devdata/env.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Agreed

Reply:
love the amicable agreement here, but I think is **needed** because this one is used by default in VSCode when running with Robocorp Code extension unless I am totally mistaken

Reply:
Yeah env.json is also needed for the rcc users, so I'd keep it.

Reply:
Isn't this _env.json_ a duplicate of _env-for-producer.json_ one? And VSCode automatically detects now the Work Items, it doesn't require any _env.json_ at all. ([details](https://github.com/robocorp/template-producer-consumer/pull/2#discussion_r899199875))

Reply:
Okay dokey